Sources/Commands/SwiftTestTool.swift

```diff
@@ -68,6 +131,9 @@ public class TestToolOptions: ToolOptions {
 
         if shouldRunInParallel {
             return .runParallel
+            
+        } else if numberOfWorkers != nil {
```

Comment:
I can remove it and turn on the shouldRunInParallel flag on the --num-workers argument definition. Does that sound better?

I made the condition instead because I thought it made the intention clearer. But the other solution works as well